### ISSUE TITLE
UMEP Plugin

### ISSUE THREAD

**Issue by ragini9m** [UMEP-dev/UMEP#642]
**Describe the bug**
Pops up when I go to SVF(sky view factor) and other pre-processing and processing tools: This tool has moved to UMEP for processing, available via the QGIS Plugin Manager. Visit our online manual for more information.

**To Reproduce**
1. Go to 'UMEP'
2. Click on 'Pre-processing'
3. Scroll down to 'Urban Geometry'
4. Select from SVF or Wall hieght
5. See error

**Desktop (please complete the following information):**
 - Windows 11
 - QGIS Version [e.g. 3.18]
 - UMEP Version [e.g. 3.16.2]

**Screenshots**
![WhatsApp Image 2024-09-04 at 12 13 51_60a99085](https://github.com/user-attachments/assets/e0f26e70-93e5-4012-82fc-956f9797a035)

**Data**
Daily Shading tutorial.

**Additional context**

**Comment by ragini9m** [comment]
![image](https://github.com/user-attachments/assets/4e768486-58c0-442d-b163-f8e60c243338)
Have the Plugins installed but still doesn't work

**Comment by ragini9m** [comment]
QGIS version is 3.34.10 and LTR

**Comment by nilswallenberg** [comment]
Hi,

Have you checked your Processing toolbox (lower right corner).

Cheers,
Nils

**Comment by ragini9m** [comment]
Hello,

The following is a snap from the toolbox
[image: image.png]

However, I can't find the following which I have taken from a youtube
tutorial
[image: image.png]

Thank you so much for the response, really means a lot to me and hope
to hear from you soon.

Regards,
Ragini Mishra

**Comment by biglimp** [comment]
![image](https://github.com/user-attachments/assets/6f4bb352-d9a3-4745-805e-ef4fa375077f)

**Comment by nilswallenberg** [comment]
Can't see your attached images. 

Also, if you haven't seen them yet, we have descriptions of the tools and tutorials here:

[Descriptions](https://umep-docs.readthedocs.io/en/latest/)
[Tutorials](https://tutorial-docs.readthedocs.io/)

**Comment by ragini9m** [comment]
Yes, I have opened it exactly with the circled-out option. The pop up is
similar but doesn't have some options that are available from UMEP. I will
attach the images as soon I receive them from a colleague who has it
running.
Sorry for the delay.

**Comment by biglimp** [comment]
Was this fixed? Can I close?

**Comment by biglimp** [comment]
I will close this now. Re-open if the problem persists.
